health_check/views.py

```diff
@@ -46,3 +41,11 @@ def render_to_response_json(self, plugins, status):
             {str(p.identifier()): str(p.pretty_status()) for p in plugins},
             status=status
         )
+
+    def _run(self, plugin):
```

Comment:
self is unused and this way you will run into the GIL since you all threads need to access a thread safe object